<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-S08CUT5" name="demoTestNG.InvocationCount" tests="3" failures="0" timestamp="25 Jul 2017 18:57:11 GMT" time="3.063" errors="1">
  <testcase name="viewBalance" time="0.004" classname="demoTestNG.InvocationCount"/>
  <testcase name="login" time="3.041" classname="demoTestNG.InvocationCount">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.login() didn&#039;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.login() didn't finish within the time-out 3000
]]>
    </error>
  </testcase> <!-- login -->
  <testcase name="signUp" time="0.018" classname="demoTestNG.InvocationCount"/>
</testsuite> <!-- demoTestNG.InvocationCount -->
